doc/user/pbr.rst

```diff
@@ -161,6 +161,26 @@ specified in the rule are also applied to the packet.
    VLAN-matching facilities,
    so this field will be ignored unless other dataplane providers are used.
 
+.. clicmd:: set nexthop-group NAME
+
+   Action:
+   forward the packet using nexthop-group NAME.
+
+.. clicmd:: set nexthop [A.B.C.D|X:X::X:XX|drop] [interface] [nexthop-vrf NAME]
```

Comment:
Can't it be `blackhole` or `null`, or `reject` to be consistent with other namings we have per next-hop?

Reply:
Agreed, changed "drop" to "blackhole"